### PR TITLE
Refactoring after release 2 feedback

### PR DESCRIPTION
**Pull request by n2chao**

### CODE REVIEW

File: app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt [hunk 1]
```diff
@@ -51,6 +51,9 @@ class MainActivity : SimpleActivity(), PreviewListener, PhotoProcessor.MediaSave
     private val MINIMUM_TIME_BETWEEN_UPDATES: Long = 10000L //milliseconds
     private val MINIMUM_DISTANCE_CHANGE_FOR_UPDATES: Float = 100f //meters
     private val GPS_REQUEST_CODE = 101
```
Comment (Earthii) [inline app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt]: Nice magic number refactoring

File: app/src/main/kotlin/com/simplemobiletools/camera/activities/SmileDetectionActivity.kt [hunk 2]
```diff
@@ -37,22 +37,34 @@ import kotlinx.android.synthetic.main.activity_smile.*
 class SmileDetectionActivity : AppCompatActivity(){
     private val TAG = "FaceTracker"
 
+    private val CAMERA_FPS = 30.0f
+    private val RC_HANDLE_GMS = 9001
+    private val RC_HANDLE_CAMERA_PERM = 2
+    private val PROGRESS_BAR_FULL = 100
```
Comment (Earthii) [inline app/src/main/kotlin/com/simplemobiletools/camera/activities/SmileDetectionActivity.kt]: Nice, this follows our code convention better

File: app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt (hunk 1 above)
Comment (StevenHaa) [reply app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt]: nice refactoring of using these variables in capitals to show its a constant!

File: app/src/main/kotlin/com/simplemobiletools/camera/activities/SmileDetectionActivity.kt [hunk 3]
```diff
@@ -162,8 +168,8 @@ class SmileDetectionActivity : AppCompatActivity(){
     fun takePicture(){
         mCameraSource!!.takePicture(null, CameraSource.PictureCallback { bytes ->
             try {
-                var i = this.intent
-                startActivity(i)
+                var intent = this.intent
```
Comment (StevenHaa) [inline app/src/main/kotlin/com/simplemobiletools/camera/activities/SmileDetectionActivity.kt]: better clarity for sure, usually i is used for index

**Comment by Ea10** [comment]
#133

**Pull request merged**